apps/camera/js/views/controls.js

```diff
@@ -245,11 +250,19 @@ module.exports = View.extend({
     this.unset(key ? key + '-enabled' : 'enabled');
   },
 
+  localize: function() {
+    for (var el in this.elsL10n) {
+      this.els[el].setAttribute('data-l10n-id', this.elsL10n[el]);
```

Comment:
I'm not sure I understand the point of what this function is doing. Aren't the `data-l10n-id` attributes already hard-coded in the `template` below to the same values already defined in `this.elsL10n` above? I also don't see anywhere that `this.elsL10n` gets changed.